packages/mobile/components/TopBar.svelte

```diff
@@ -0,0 +1,13 @@
+<script>
+    import { AccountSwitcher, AccountActionsButton } from './'
+
+    export let profileInitial = 'A'
+</script>
+
+<div class="flex flex-row justify-between">
+    <button class="w-10 h-10 relative flex items-center justify-center rounded-full bg-blue-500 leading-100">
```

Comment:
Dont we have a component we can use for this? Desktop also has the initial bubble in the bottom left, so perhaps we can reuse it or create our own according to the style specs we have on mobile

Reply:
I searched in the Sidebar but just found the button. I would create a component for mobile